nengo/utils/ipython.py

```diff
@@ -9,18 +9,18 @@
     from IPython import get_ipython
     from IPython.display import HTML
 
-    if IPython.version_info[0] <= 3:
+    if IPython.version_info[0] <= 3:  # pragma: no cover
         from IPython.nbconvert import PythonExporter
-    else:
+    else:  # pragma: no cover
```

Comment:
We should only have to "no cover" one of these two branches (I'm guessing the one for older IPython).